grpc/nodejs/BUILD

```diff
@@ -71,24 +71,22 @@ genrule(
     #    Output typescript to (you guessed it) the genrule output directory
     #    Set the .proto file relative path to the WORKSPACE this command is being run from
     #    Use the .proto files found in the :proto-raw-buffers filegroup as inputs.
-    cmd = "ln -s $(execpath //grpc/nodejs:protoc-gen-ts).runfiles/npm/node_modules ./node_modules ;\
-        $(execpath //grpc/nodejs:grpc_tools_node_protoc) \
+    cmd = "$(execpath //grpc/nodejs:grpc_tools_node_protoc) \
             --plugin='protoc-gen-ts=$(rootpath @npm//:node_modules/grpc_tools_node_protoc_ts/bin/protoc-gen-ts)' \
             --js_out='import_style=commonjs,binary:./$(@D)/' \
             --grpc_out='grpc_js:./$(@D)/' \
             --ts_out='grpc_js:./$(@D)/' \
-            --proto_path=`dirname $(execpath //:WORKSPACE)` \
```

Comment:
`execpath //:WORKSPACE` is only . if this is being built from the top level workspace- if it's being brought in as an external dependency, this doesn't necessarily hold- can you confirm this doesn't break when pulled in externally?

Reply:
@Phillammon confirmed: i tried pulling `protocol` from `client-java` & `client-nodejs`, none of them broke

Reply:
Also: `//grpc/nodejs:protocol` is not designed to be used as a Bazel dependency - it should only be pulled from NPM